doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -275,16 +275,18 @@ Bluetooth libraries and services
 Modem libraries
 ------------------------
 
-* :ref:`lib_location` library:
+* Updated:
 
-  * Added:
+  * :ref:`sms_readme` library:
 
-    * Support for P-GPS data retrieval from an external source, implemented separately by the application.
-      Enabled by setting the :kconfig:option:`CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL` option.
-      The library triggers a :c:enum:`LOCATION_EVT_GNSS_PREDICTION_REQUEST` event when assistance is needed.
+    * Added handling for SMS client unregistration notification from the modem.
+      When the notification is received, the library re-registers the SMS client automatically.
 
-  * Updated:
+  * :ref:`lib_location` library:
```

Comment:
We seem to have another entry for this library (line 343) and that is in the wrong place. Should we move it here as well?